recipes/cocotb/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "cocotb" %}
+{% set version = "1.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cocotb-{{ version }}.tar.gz
+  sha256: 959892eb94bd0b3ff40e0fca51d33a3936416deb853e2bac4f7f766b40002650 
+  patches:
+    - patches/0001-remove-static-stdc.patch
+
+build:
+  number: 0
+  skip: true  # [win or py<310]
+  entry_points:
+    - cocotb-config = cocotb.config:main
+  run_exports:
+    - {{ pin_subpackage("cocotb", max_pin="x") }}
+  script:
+    - {{ PYTHON }} -m pip install -U --no-deps . -vvv
```

```suggestion
    - {{ PYTHON }} -m pip install --no-deps . -vvv
```
Why do you specify `--upgrade`?